### DISCUSSION TITLE
I have a suggestion about an SSISDB catalog panel.

### DISCUSSION THREAD

**Discussion by mpaivant** [trimble-oss/dba-dash#1041] [Ideas]
You could add an SSISDB catalog panel to your project, it would help a lot.

![Screenshot_1](https://github.com/user-attachments/assets/3406bb84-ff0e-43fe-8f4e-f1dbd362a514)
![Screenshot_2](https://github.com/user-attachments/assets/a40e2aa6-91d9-4ead-b44c-64590db1805a)
![Screenshot_4](https://github.com/user-attachments/assets/27317460-dc2a-491d-a02f-689f57551158)

**Comment by mpaivant** [comment]
I'm not sure who it belongs to:

Andy Leonard
or
website: https://dilmsuite.com/

Some mistake sorry

Thank you very much for the solution it helps a lot

**Comment by DavidWiseman** [comment]
Hi, thanks for the suggestion.  It's been a number of years since I last used SSIS and I don't have any SSIS packages that I need to monitor.  It might be something you can add in yourself though with the [custom collections](https://dbadash.com/docs/help/custom-collections/) and [custom reports](https://dbadash.com/docs/how-to/create-custom-reports/) feature.
